File: challenges/1-produto-simples/C/emptybrain/produto.c
#include "stdio.h"

int PROD[2]; // especifica o tamanho do array

int main(){
  printf("Digite os valores: ");
  scanf("%d %d", &PROD[0], &PROD[1]);
  printf("produto: %d \n", PROD[0] * PROD[1]);
}

File: challenges/1-produto-simples/python/guilhermebrito23/produto.py
def calc_produto(value1, value2):
    '''
    Função Calcula produto
    Exemplo: calc_produto(10,3)
    output: 30
    '''
    return value1 * value2

print("Desafio #1")
try:
    first_value = int(input("Digite o primeiro valor: "))
    second_value = int(input("Digite o segundo valor: "))

    PROD = calc_produto(first_value, second_value)
    print(f'Produto: {PROD}')
except:
    print("Erro: Digite somente valores numéricos inteiros.")
